Godeps/Godeps.json

```diff
@@ -0,0 +1,60 @@
+{
+	"ImportPath": "github.com/prometheus/prometheus",
+	"GoVersion": "go1.4",
+	"Deps": [
+		{
+			"ImportPath": "code.google.com/p/goprotobuf/proto",
+			"Comment": "go.r60-152",
+			"Rev": "36be16571e14f67e114bb0af619e5de2c1591679"
+		},
+		{
+			"ImportPath": "github.com/golang/glog",
+			"Rev": "44145f04b68cf362d9c4df2182967c2275eaefed"
+		},
+		{
+			"ImportPath": "github.com/matttproud/golang_protobuf_extensions/ext",
+			"Rev": "7a864a042e844af638df17ebbabf8183dace556a"
+		},
+		{
+			"ImportPath": "github.com/miekg/dns",
+			"Rev": "6b75215519f9916839204d80413bb178b94ef769"
+		},
+		{
+			"ImportPath": "github.com/prometheus/client_golang/_vendor/goautoneg",
+			"Rev": "6fbc8ef5c2cd3ae524bfe001f92342c7caaf2b6b"
+		},
+		{
+			"ImportPath": "github.com/prometheus/client_golang/_vendor/perks/quantile",
+			"Rev": "6fbc8ef5c2cd3ae524bfe001f92342c7caaf2b6b"
+		},
+		{
+			"ImportPath": "github.com/prometheus/client_golang/extraction",
```

Comment:
This (and below) is our ewn client_golang, which we don't want to vendor, right?
Also, it looks you have vendored it to an obsolete version, which breaks travis.